README.md

```diff
@@ -47,39 +47,53 @@ When you have completed this Code Pattern, you will understand how to:
-6. [Download and move the data to HDFS](#7-download-and-move-the-data-to-hdfs)
-7. [Setup python plugins](#8-setup-python-plugins)
-8. [Launch the notebook](#9-launch-the-notebook)
-9. [Run the notebook](#10-run-the-notebook)
+## Access to HDP Platform
 
-### 1. Setup HDP platform
+The core of this code pattern is integrating Hortonworks Data Platform (HDP) and IBM DSX Local. If you do not already have an HDP cluster available for use, you will need to install one before attempting to complete the code pattern. 
 
-Please install HDP v2.6.4 by following instructions at [install_hdp](https://docs.hortonworks.com/HDPDocuments/HDP2/HDP-2.6.4/bk_command-line-installation/content/ch_getting_ready_chapter.html) to install and setup HDP cluster. 
+To install [HDP v2.6.4](https://docs.hortonworks.com/HDPDocuments/HDP2/HDP-2.6.4/index.html), please follow the [installation guide](https://docs.hortonworks.com/HDPDocuments/Ambari-2.6.1.5/bk_ambari-installation/content/ch_Getting_Ready.html) provided by Hortonworks. It first requires the installation of the [Apache Ambari](https://ambari.apache.org/) management platform which is then used to faciliate the HDP cluster installation. The Ambari Server is also required to complete a number of steps described in the following sections.
 
-After you finish installing HDP v2.6.4, you need to disable livy server's CSRF protection: go to your Ambari server UI, click the `Spark2` on the left side panel, click the `Configs` on the top, click on the `Advanced livy2-conf`
-change the variable `livy.server.csrf_protection.enable` from true to false, then restart the server.
+> Note: Ensure that your Ambari Server is configured to use `Python v2.7`.
 
-  ![](images/dsx-local-create-project-spark2-livy.png)
- 
-Since the code pattern makes sure of several python plugins (tmdbsimple, solrcloudpy, simplejson, Solr vector scoring plugin) and spark extension through solr spark connector, we strongly recommend that the components are at the right version for the pattern to run smoothly. 
-	
-	1. HDP is at v2.6.4
-	1. Python version is at 2.7
-	1. Spark version is at 2.2
+## Install HDP Cluster services
+
+Once your HDP cluster is deployed, at a minimum, install the following services as listed in this Ambari Server UI screenshot:
+
+![](images/ambari-services.png)
+
+> Note: 
+> 1. This code pattern requires that version `2.2.0` of the `Spark2` service be installed.
+> 2. Details on how to install and configure the `Solr` service are discussed in the [Setup HDP Search](#1-set-up-hdp-search) section below.
+
+## Disable Apache Livy CSRF protection
+
+From your Ambari Server UI, disable the Apache Livy server's CSRF proctection by changing the `livy.server.csrf_protection.enable` variable to false, as shown here:
+
+![](images/dsx-local-create-project-spark2-livy.png)
+
+To navigate to this panel, click on the `Spark2` service, then the `Configs` tab, and then open the `Advanced livy2-conf` section.
```

Comment:
nit pick for readability here, make the thing that the user does first, appear first. logically, this should go earlier...

```
We can disable the Apache Livy server's CSRF protection from the Ambari Server UI. To do this, in the console click on the `Spark2` service, then the `Configs` tab, and then open the `Advanced livy2-conf` section. Now we can choose to disable the Livy server's CSRF protection by changing `livy.server.csrf_protection.enable` to `false`. See the example below.
```